diff -up dooble-2022.06.15/dooble.pro.1~ dooble-2022.06.15/dooble.pro
--- dooble-2022.06.15/dooble.pro.1~	2022-06-15 15:20:58.000000000 +0200
+++ dooble-2022.06.15/dooble.pro	2022-09-14 18:46:30.248876421 +0200
@@ -252,7 +252,6 @@ QMAKE_CXXFLAGS_RELEASE += -O3 \
                           -Wcast-align \
                           -Wcast-qual \
                           -Wdouble-promotion \
-                          -Werror \
                           -Wextra \
                           -Wformat=2 \
                           -Woverloaded-virtual \
@@ -297,7 +296,6 @@ QMAKE_CXXFLAGS_RELEASE += -O3 \
                           -Wcast-align \
                           -Wcast-qual \
                           -Wdouble-promotion \
-                          -Werror \
                           -Wextra \
                           -Wformat-overflow=2 \
 			  -Wformat-signedness \
diff -up dooble-2022.06.15/Source/dooble.cc.1~ dooble-2022.06.15/Source/dooble.cc
--- dooble-2022.06.15/Source/dooble.cc.1~	2022-06-15 15:20:58.000000000 +0200
+++ dooble-2022.06.15/Source/dooble.cc	2022-09-14 18:45:42.838810294 +0200
@@ -27,6 +27,7 @@
 
 #include <QFileDialog>
 #include <QMessageBox>
+#include <QPainter>
 #include <QPointer>
 #include <QPrintDialog>
 #include <QPrintPreviewDialog>
diff -up dooble-2022.06.15/Source/dooble_tab_bar.cc.1~ dooble-2022.06.15/Source/dooble_tab_bar.cc
--- dooble-2022.06.15/Source/dooble_tab_bar.cc.1~	2022-06-15 15:20:58.000000000 +0200
+++ dooble-2022.06.15/Source/dooble_tab_bar.cc	2022-09-14 18:45:42.839810296 +0200
@@ -26,6 +26,7 @@
 */
 
 #include <QWebEngineSettings>
+#include <QActionGroup>
 #include <QtMath>
 
 #include <algorithm>
